src/main/java/com/heatheryou/httpserver/handler/HeadHandler.java

```diff
@@ -0,0 +1,24 @@
+package com.heatheryou.httpserver.handler;
+
+import com.heatheryou.httpserver.Request;
+import com.heatheryou.httpserver.Response;
+import com.heatheryou.httpserver.ResponseBuilder;
+import com.heatheryou.httpserver.constants.CharacterSet;
+import com.heatheryou.httpserver.constants.EntityHeader;
+
+import static com.heatheryou.httpserver.constants.CharacterSet.EMPTY;
+import static com.heatheryou.httpserver.constants.StatusLine.STATUS_CODE_200;
+
+public class HeadHandler implements IHandler {
```

Comment:
Currently there's no difference between how `HeadHandler` behaves vs `GetHandler` so it seems premature to separate them out already.

Reply:
Makes sense, removed it for now!